File: src/Pisaz.Backend.API/Models/ClientModels/Address.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace Pisaz.Backend.API.Models.ClientModels
{
    public class Address
    {
        [JsonIgnore]
        public int ID { get; set; }
        
        public required string Province { get; set; }
        public required string Remainder { get; set; }
    }
}

File: src/Pisaz.Frontend/lib/models/address.dart
class Address {
  final String province;
  final String remainder;

  Address({required this.province, required this.remainder});

  factory Address.fromJson(Map<String, dynamic> json) {
    return Address(
        province: json['province'] as String,
        remainder: json['remainder'] as String);
  }
}
